.github/workflows/fix-linter-hints.yml

```diff
@@ -55,3 +55,4 @@ jobs:
           committer: Dr. Lint-a-lot <garybot2@graylog.com>
           commit-message: Running lint --fix
           delete-branch: true
+          token: ${{ secrets.PAT_GRAYLOG_PROJECT_INTERNAL_WORKFLOW_RW }}
```

Comment:
I think that won't work, the token lacks permissions to create PRs.
I can move that job to another repo if you like.

Reply:
How about creating a single job in graylog-project-internal that updates the `yarn.lock` files in all repos in a project manifest?

Reply:
How about this then?

https://github.com/Graylog2/graylog2-server/pull/17657